### PR TITLE
New branch

### PR DESCRIPTION
**Pull request by ncampbell89**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "start": "node ./bin/www",
+    "server": "nodemon ./bin/www",
+    "client-install": "npm install --prefix client",
+    "client": "npm start --prefix client",
+    "dev": "concurrently \"npm run server\" \"npm run client\""
+  },
+  "engines": {
+    "node": "10.10.0"
+  },
+  "author": "Imran Sayed",
```
Comment (bonnieli) [inline package.json]: whose Imran haha. maybe change this to Manpreet and Nicole

Comment (bonnieli) [reply package.json]: it looks like this is running concurrently server and dev. please write an instructional README.md file :)

File: client/src/App.js [hunk 2]
```diff
@@ -1,19 +1,40 @@
...
-                </Switch>
-            </Router>
+            <Provider store={store}>
+                <Router>
+                    <div>
+                        {token ? <Header /> : ''}
+                        <Switch>
+                            <Route exact path="/" component={Login} />
+                            <Route exact path="/signup" component={SignUp} />
+                            <Route exact path="/forgotpswd" component={PswdRecover} />
```
Comment (bonnieli) [inline client/src/App.js]: suggestion to rename these routes to camelcase like `forgotPassword`

File: client/src/App.js [hunk 3]
```diff
@@ -1,19 +1,40 @@
...
+                            <Route exact path="/newfund" component={NewFund} />
+                            <Route exact path="/messages" component={Messages} />
+                            <Route exact path="/myfunds" component={MyFunds} />
+                            <Route exact path="/profiles" component={Profiles} />
+                            <Route exact path="/myprofile" component={MyProfile} />
```
Comment (bonnieli) [inline client/src/App.js]: `/profile`

File: client/src/App.js [hunk 4]
```diff
@@ -1,19 +1,40 @@
...
+                <Router>
+                    <div>
+                        {token ? <Header /> : ''}
+                        <Switch>
+                            <Route exact path="/" component={Login} />
+                            <Route exact path="/signup" component={SignUp} />
+                            <Route exact path="/forgotpswd" component={PswdRecover} />
+                            <Route exact path="/newfund" component={NewFund} />
+                            <Route exact path="/messages" component={Messages} />
+                            <Route exact path="/myfunds" component={MyFunds} />
```
Comment (bonnieli) [inline client/src/App.js]: `/funds`

File: client/src/components/NewFund.js [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+        } else {
+            // if file not selected throw error
+            this.ocShowAlert( 'Please upload file', 'red' );
+        }
+      };
+
+	// ShowAlert Function
+	ocShowAlert = ( message, background = '#3089cf' ) => {       
+		let alertContainer = document.querySelector( '#oc-alert-container' ),
+			alertEl = document.createElement( 'div' ),
```
Comment (bonnieli) [inline client/src/components/NewFund.js:123]: avoid using manipulating the DOM when using react. you can create a new component for this alert dialog

**Review by ManpreetBhuller — APPROVED** [review]

**Pull request closed without merging**